libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -51,32 +51,39 @@ export type PreloadStrategy = Partial<{
 /**
  *
  */
-export type BroadcastArg = {
-  account: Account;
+export type BroadcastArg<A extends Account> = {
+  account: A;
   signedOperation: SignedOperation;
 };
 
 /**
  *
  */
-export type SignOperationArg0<T> = {
-  account: Account;
+export type SignOperationArg0<T extends TransactionCommon, A extends Account> = {
+  account: A;
   transaction: T;
   deviceId: DeviceId;
 };
 
 /**
  *
  */
-export type SignOperationFnSignature<T> = (
-  arg0: SignOperationArg0<T>,
+export type SignOperationFnSignature<T extends TransactionCommon, A extends Account> = (
+  arg0: SignOperationArg0<T, A>,
 ) => Observable<SignOperationEvent>;
 
-export type BroadcastFnSignature = (arg0: BroadcastArg) => Promise<Operation>;
+export type BroadcastFnSignature<A extends Account = Account> = (
+  arg0: BroadcastArg<A>,
+) => Promise<Operation>;
 
-export type Bridge<T extends TransactionCommon> = {
+export type Bridge<
+  T extends TransactionCommon,
+  U extends TransactionStatusCommon = TransactionStatusCommon,
```

Comment:
I wonder, if it is not best to define `U` as the last generic parameter to benefit from the default value.
`TransactionStatus` seems to be the least override type in CoinModules.

Reply:
Yup, was thinking about it as well, I'm just worried that `TransactionStatus` will probably always be omitted, and might not be reflected correctly once reaching userland (which is the original reason of this whole PR, because I need the UI to know what kind of keys are in the `TransactionStatus` of the EVM bridge, and I didn't want to cast it 🤣), but maybe just enforcing it at the `js.ts` level would be enough for that. Will see how hard it is to change the order 👌

Reply:
Done, order should be better now 👍